docker/llm/serving/cpu/docker/entrypoint.sh

```diff
@@ -101,7 +102,7 @@ if [ "$#" == 0 ]; then
   exec /usr/bin/tini -s -- "bash"
 else
   # Parse command-line options
-  options=$(getopt -o "m:h" --long "mode:,help" -n "$0" -- "$@")
+  options=$(getopt -o "m:hw:" --long "mode:,help" -n "$0" -- "$@")
```

Comment:
Does "worker" need to be set in --long flag here?

Reply:
fixed